client/me/security-2fa-backup-codes/password-prompt.jsx

```diff
@@ -0,0 +1,68 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
We are only using `Component`, so perhaps let's just import that instead of all of `React`.